test.py

```diff
@@ -113,6 +113,16 @@ def test_dada2_learn_errors():
         "bio/dada2/learn-errors",
         ["snakemake", "--cores", "1", "--use-conda", "results/dada2/model_1.RDS"],
     )
+def test_dada2_filter_trim_se():
+    run(
+        "bio/dada2/filter-trim",
+        ["snakemake", "--cores", "1", "filtered-se/a.1.fastq", "--use-conda", "-F"]
+    )
+def test_dada2_filter_trim_pe():
+    run(
+        "bio/dada2/filter-trim",
+        ["snakemake", "--cores", "1", "filtered-pe/a.1.fastq", "--use-conda", "-F"]
+    )
```

Comment:
I guess it's mostly for readability, but usually you would add an empty row before and after each function definition in Python (and this might incidentally help with the merge conflicts, although I'm not sure):
```suggestion

def test_dada2_filter_trim_se():
    run(
        "bio/dada2/filter-trim",
        ["snakemake", "--cores", "1", "filtered-se/a.1.fastq", "--use-conda", "-F"]
    )

def test_dada2_filter_trim_pe():
    run(
        "bio/dada2/filter-trim",
        ["snakemake", "--cores", "1", "filtered-pe/a.1.fastq", "--use-conda", "-F"]
    )

```